### PR TITLE
STRATEGY: add liquidity maker

### PR DESCRIPTION
**Pull request by c9s**

### CODE REVIEW

**Review by kbearXD — APPROVED** [review]

File: pkg/strategy/liquiditymaker/generator.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	side types.SideType, totalAmount, startPrice, endPrice fixedpoint.Value, numLayers int, scale bbgo.Scale,
+) (orders []types.SubmitOrder) {
+
+	if g.logger == nil {
+		logger := log.New()
+		logger.SetLevel(log.ErrorLevel)
+		g.logger = logger
+	}
+
+	layerSpread := endPrice.Sub(startPrice).Div(fixedpoint.NewFromInt(int64(numLayers)))
```
Comment (gx578007) [inline pkg/strategy/liquiditymaker/generator.go:49]: Divided by "numLayer-1"?

File: pkg/strategy/scmaker/strategy.go [hunk 2]
```diff
@@ -5,22 +5,18 @@ import (
...
 	"github.com/c9s/bbgo/pkg/fixedpoint"
 	. "github.com/c9s/bbgo/pkg/indicator/v2"
-	"github.com/c9s/bbgo/pkg/risk/riskcontrol"
 	"github.com/c9s/bbgo/pkg/strategy/common"
 	"github.com/c9s/bbgo/pkg/types"
 )
 
 const ID = "scmaker"
 
-var ten = fixedpoint.NewFromInt(10)
```
Comment (gx578007) [inline pkg/strategy/scmaker/strategy.go:22]: Is this expected to touch scmaker?

File: pkg/strategy/liquiditymaker/generator.go (hunk 1 above)
Comment (c9s) [reply pkg/strategy/liquiditymaker/generator.go]: You're right LOL, fixed

File: pkg/strategy/scmaker/strategy.go (hunk 2 above)
Comment (c9s) [reply pkg/strategy/scmaker/strategy.go:22]: yes, I cleaned up the scmaker and then copy scmaker to liquiditymaker

**Pull request merged**